sdk/batch/arm-batch/test/batchManagementClient.spec.ts

```diff
@@ -0,0 +1,577 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { BatchManagementClient, BatchManagementModels } from "../src/batchManagementClient";
+import { describe, beforeEach } from "mocha";
+import { assert } from "chai";
+import * as dotenv from "dotenv";
+import * as util from "util";
+import * as fs from "fs";
+import { ServiceClient, WebResource } from "@azure/core-http";
```

Comment:
@joheredi Is it ok to use `core-http` in tests for a track1 package?

Reply:
They may be compatible, however I think test should use `ms-rest-js for` consistency